### DISCUSSION TITLE
Menu cursor spins after holding left click, moving it in a circle and releasing

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#22550] [Q&A] [answered]
Issue #16114 (Menu cursor spins after playing beatmap) can be recreated anywhere you can see the menu cursor, not just beatmaps with spinners at the end. I was able to recreate this on a Windows computer and an Android phone.

https://user-images.githubusercontent.com/124777318/217500399-d101826b-edf2-4cb3-aded-3cd8b5fe4bd5.mp4

Used phone: Xiaomi Redmi Note 9
Osu! client version: 2023.207.0

**Comment by Cootz** [comment, marked as answer]
It's a feature. 
If you wanna turn it off uncheck this:
![image](https://user-images.githubusercontent.com/50776304/217506212-33688ef2-59f1-4861-9286-ba896f7d17f8.png)

**Reply by ghost** [reply]
I'm not talking about it just turning, watch the whole video and maybe check the issue #16114 's video.

**Reply by Cootz** [reply]
This [issue](https://github.com/ppy/osu/issues/16114) is about mouse somehow using the rotation from a spinner after a play to spin the cursor. Not about spinning cursor itself

**Reply by ghost** [reply]
And this discussion page is about the issue not being limited to spinners. You can see the cursor spinning at the end just like #16114

**Reply by Theighlin** [reply]
That's a feature, you're doing it on purpose. The issue you keep linking is about it happening involuntarily after a spinner.
